packages/editor/package.json

```diff
@@ -13,7 +13,8 @@
     "@theme-ui/style-guide": "^0.2.49",
     "color": "^3.1.2",
     "react-color": "^2.17.3",
-    "reakit": "^1.0.0-beta.6"
+    "reakit": "^1.0.0-beta.6",
+    "tinycolor2": "^1.4.1"
```

Comment:
This is the same lib that `react-color` uses, right?

Reply:
Yeah it is 👍